StaticAnalyzer/views/ios/binary_analysis.py

```diff
@@ -381,7 +381,7 @@ def class_dump(tools_dir, bin_path, app_dir, bin_type):
                 jtool_bin = settings.JTOOL_BINARY
             else:
                 jtool_bin = os.path.join(tools_dir, 'jtool.ELF64')
-            os.chmod(jtool_bin, 0o777)
+            os.chmod(jtool_bin, 0o744)
```

Comment:
Here

Reply:
in my test yes but it means that the user who do the install is the same who launch MobSF 
774 can be a safe alternative